docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md

```diff
@@ -108,10 +108,14 @@ To import an updated version of the same file (that is, having the same file nam
 yb-voyager import data file --data-dir /dir/data-dir --file-table-map 'orders.csv:orders' ...
 ```
 
-After new rows are added to `orders.csv`, use the following command to load them with `--start-clean`:
+After new rows are added to `orders.csv`, use the following command to load them with the flags `--start-clean` and `--enable-upsert`.
+Note: Ensure that tables on the target YugabyteDB database do not have secondary indexes. If a table has secondary indexes, using `--enable-upsert true` may lead to corruption of the indexes.
```

Comment:
Can this be made more prominent? It's essentially a warning, so want it to catch the user's attention - perhaps a box?